package.json

```diff
@@ -41,6 +41,7 @@
     "test:common": "mocha --exit --require ./test/setup/mocha.js src/common/**/*.mocha.{ts,tsx}",
     "test:client": "mocha --exit --require ./test/setup/mocha.js src/client/**/*.mocha.{ts,tsx}",
     "test:server": "mocha --exit --require ./test/setup/mocha.js src/server/**/*.mocha.ts",
+    "test:size": "size-limit",
```

Comment:
Do we want to run `size-limit` inside `npm test`?

Reply:
I would like. I want it to fail when the declared limit is exceeded.